server/controllers/projects/getProject.js

```diff
@@ -5,8 +5,19 @@ exports.getProject = (req, res, next) => {
   const { projectId } = req.params;
   selectProjectDetails(projectId)
     .then((result) => {
+      const {
+        id,
+        name,
+        description,
+        userNames,
+      } = refactorData(result.rows);
       const projectsData = {
-        data: refactorData(result.rows),
+        data: {
+          id,
+          name,
+          description,
+          userNames,
```

Comment:
@aminalakhsham, the data structure changed. Sorry, but we had to change ```user_id``` to ```userNames``` which is an array of objects: ```[{user_id: 1, full_name: 'Amin al Akhsham'}, {user_id: 2 , full_name: 'Ahmed Shatat'} ]```

Reply:
this is the main problem with edit project page>>>
I make PR to overcome this

Reply:
Good, thanks Ayman